release_docs/RELEASE.txt

```diff
@@ -50,7 +50,7 @@ New Features
     - Renamed the option: HDF5_ENABLE_Z_LIB_SUPPORT
 
       The option has been renamed to HDF5_ENABLE_ZLIB_SUPPORT to be consistent
-      with the naming of other options.
+      with the naming of other options. Also, the option defaults to OFF.
```

Comment:
Definitely want to make sure this is very well called out for the next release

Reply:
Absolutely